pkg/installer/src/tpmmgr/tpmmgr.go

```diff
@@ -0,0 +1,1279 @@
+// Copyright (c) 2024 Zededa, Inc.
```

Comment:
Is this file a copy of `pillar/cmd/tpmmgr/tpmmgr.go`?

Reply:
Almost. I trimmed it down a lot, just to what we need in installer. And to run standalone.

This all should be replaced soon enough by @jsfakian, I believe.